pkg/asset/store.go

```diff
@@ -1,10 +1,19 @@
 package asset
 
 import (
+	"encoding/json"
 	"github.com/pkg/errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	stateFileName = ".assetState.json"
```

Comment:
can we not camel case. maybe this: `.openshift_install_state.json`

Reply:
.openshift_install_state.json now.